### DISCUSSION TITLE
Átszemélyesítés 5.13.1.

### DISCUSSION THREAD

**Discussion by PereszlenyiArpadECR-Trade** [nav-gov-hu/eRECEIPT#30] [Kérdések és válaszok]
Az 1.0-ás fejlesztői dokumentáció 5.13.1. fejezetében szerepel, hogy az átszemélyesítés indításakor meg kell küldeni a legutóbbi recordCounter adatot, ami akár Napi forgalmi jelentés is lehet (ha jól értelmezem), viszont a törlés során ennek az utolsónak meg kell maradnia, azaz átszemélyesítés után az új tulajdonoshoz kerülhet az előző tulajdonos forgalmi adata (utolsó Napi forgalmi jelentése). Ennek kezelésére javasolt egy általános megoldás beiktatása, hogy ne lehessen az utolsó recordCounter aóügyi adatot tartalmazó bizonylat/bejegyzés.

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
